src/components/test/Feed.test.tsx

```diff
@@ -6,12 +6,23 @@ import { componentWithStore, createMockStore } from "../../test/testhelpers";
 import { getPreFabSocialGraph } from "../../test/testGraphs";
 import { getPrefabProfile } from "../../test/testProfiles";
 import { FeedItem } from "../../utilities/types";
+import { waitFor } from "@testing-library/react";
 
-const userId = getPrefabProfile(0).fromId;
+const userProfile = getPrefabProfile(0);
+const displayProfile = getPrefabProfile(3);
 const feedItems: FeedItem[] = getPrefabFeed();
 const graphs = getPreFabSocialGraph();
 const initialState = {
-  user: { id: userId },
+  user: {
+    id: userProfile.fromId,
+    displayId: displayProfile.fromId as string | undefined,
+  },
+  profiles: {
+    profiles: {
```

Comment:
why profiles.profiles seems repetitive?

Reply:
This has been that way in the profileFeed for a while (the first `profile` is implicit. it's probably worth changing, but it should be done in a separate story since it touches a lot of stuff.

Reply:
Yeah I noticed that. are we forced to continue with that pattern?